standard/expressions.md

```diff
@@ -3449,16 +3457,22 @@ The predefined subtraction operators are listed below. The operators all subtrac
   >    static void Main() {
   >       D cd1 = new D(C.M1);
   >       D cd2 = new D(C.M2);
-  >       D cd3 = cd1 + cd2 + cd2 + cd1; // M1 + M2 + M2 + M1
-  >       cd3 -= cd1; // => M1 + M2 + M2
-  >       cd3 = cd1 + cd2 + cd2 + cd1; // M1 + M2 + M2 + M1
-  >       cd3 -= cd1 + cd2; // => M2 + M1
-  >       cd3 = cd1 + cd2 + cd2 + cd1; // M1 + M2 + M2 + M1
-  >       cd3 -= cd2 + cd2; // => M1 + M1
-  >       cd3 = cd1 + cd2 + cd2 + cd1; // M1 + M2 + M2 + M1
-  >       cd3 -= cd2 + cd1; // => M1 + M2
-  >       cd3 = cd1 + cd2 + cd2 + cd1; // M1 + M2 + M2 + M1
-  >       cd3 -= cd1 + cd1; // => M1 + M2 + M2 + M1
+  >       D cd3 = null;                   // null
+  >       cd3 -= cd1;                     // => null
+  >       cd3 = cd1 + cd2 + cd2 + cd1;    // M1 + M2 + M2 + M1
```

Comment:
I suspect all of this would be clearer if we never modify any variables, instead declaring a new variable on each line.
That way we wouldn't need to reassign cd3 the same value multiple times.

Let me know if you want me to write out explicitly what I mean.